### PR TITLE
Update toddow tooltip and logic to use new aggregation endpoint

### PR DESCRIPTION
**Pull request by moradology**

### CODE REVIEW

**Comment by moradology** [comment]
it

**Comment by ddohler** [comment]
Looks like people sleep in on Sundays!

**Comment by kshepard** [comment]
That's really cool looking with lots of data. Love how it shows some clear patterns in the data.

File: web/app/scripts/toddow/toddow-directive.js [hunk 1]
```diff
@@ -148,6 +147,6 @@
     }
 
     angular.module('driver.toddow')
-    .directive('driverToddow', ToDDoW);
+      .directive('driverToddow', ToDDoW);
```
Comment (kshepard) [inline web/app/scripts/toddow/toddow-directive.js]: We have these two lines aligned in all the other files; might as well keep it consistent.

**Comment by kshepard** [comment]
The ToDDoW is on the map page now too, so the map controller will need to change accordingly. +1 after that.

**Comment by moradology** [comment]
screen shot 2015-10-15 at 3 34 58 pm 2

**Comment by kshepard** [comment]
:+1:

**Comment by moradology** [comment]
(waiting for travis just to be sure)

**Pull request merged**